code/datums/ruins/space.dm

```diff
@@ -177,7 +177,7 @@
 	description = "Until very recently this pirate radio station was used to harangue local space stations over a variety of perceived \"ethics violations\". \
 	It seems like someone finally got sick of it, but the equipment still works."
 
-/* shiptest: derelict is too big. we're commenting it out for now
+/* voidcrew: derelict is too big. we're commenting it out for now
```

Comment:
In this instance, renaming shiptest to voidcrew implies we made this decision, but we didn't. We shouldn't blame ourselves for actions that are not ours.